pubtools/_quay/clear_repo.py

```diff
@@ -123,13 +125,15 @@ def clear_repositories(repositories, settings):
     signers = settings["signers"].split(",")
     signer_configs = settings["signer_configs"].split(",")
     outdated_manifests = []
+    # TOFIX: mad can be None
     for repo, tag, mad in existing_manifests:
-        outdated_manifests.append((mad.digest, tag, repo))
+        outdated_manifests.append((mad.digest, tag, repo))  # type: ignore
 
     for n, signer in enumerate(signers):
         signercls = SIGNER_BY_LABEL[signer]
         _signer = signercls(config_file=signer_configs[n], settings=settings)
-        _signer.remove_signatures(outdated_manifests, _exclude=[])
+        # TOFIX: outdated_manfests sould be list?, also exclude is bool?
```

Comment:
outdated_manifests is List[Tuple[str, str, str]] and exclude is Optional[List[Tuple[str, str, str]]] _remove_signatures method signature in abstract class SignerWrapper needs to be updated